src/test/java/bgu/spl/mics/SquadTest.java

```diff
@@ -38,7 +33,7 @@ public void testSingleton(){
 
     @Test
     public void testLoad(){
-        assertTrue(squad.getAgentsNames(agentsSerials).equals(agentsNames), "names in squad doesn't match inserted agents' serials");
+        assertEquals(squad.getAgentsNames(agentsSerials), agentsNames, "names in squad doesn't match inserted agents' serials");
```

Comment:
I don't really trust it, because I didn't see it knows to compare arrays... did you see  function with Object[]?